releases/TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Setup
```

Comment:
we don't need a template in this repo. we've been trying to centralize release information in the axelar-contract-deployments repo where a release doc should be created for `ampd`. Often, our releases are inter-dependent, so I think it makes it easier to update all the info in one repo around core, wasm/external contracts, ampd.